sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -18,10 +18,7 @@
 package test.org.apache.spark.sql;
 
 import java.io.Serializable;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I generally wouldn't bother collapsing these imports. But don't change it unless you need to make other changes for this PR.